hana/lib/HANAService.js

```diff
@@ -908,6 +909,7 @@ class HANAService extends SQLService {
       // Not encoded string with CESU-8 or some UTF-8 except a surrogate pair at "base64_decode" function
       Binary: e => `HEXTOBIN(${e})`,
       Boolean: e => `CASE WHEN ${e} = 'true' THEN TRUE WHEN ${e} = 'false' THEN FALSE END`,
+      Vector: e => `TO_REAL_VECTOR(${e})`,
```

Comment:
We might also  send the binary variant over the wire (less bytes), but I'm fine to just use `TO_REAL_VECTOR` for now.